README.md

```diff
@@ -32,14 +32,15 @@ This will add [PCT][] tasks and types to your project and integrate
 The following are a list of enhancements beyond the original [grabl] plugin
 
  - adds a dependency on [PCT] 211
- - adds a dependency on Google gson 2.8.0 which is required by PCT
-   ABLUnit task
+ - adds a dependency on Google gson 2.8.0 which is required by PCT ABLUnit task
  - adds a backup database task
  - adds the ability to run ABL code as a task
  - adds ability to create a database
+ - adds ability to create procedure libraries
 
```

Comment:
Are we not including the creation of oear?

Reply:
we can add that